governance/src/main/java/org/apache/servicecomb/governance/marker/GovernanceRequest.java

```diff
@@ -102,4 +109,12 @@ public String getServiceName() {
   public void setServiceName(String serviceName) {
     this.serviceName = serviceName;
   }
+
+  public Object getSourceRequest() {
+    return sourceRequest;
+  }
+
+  public void setSourceRequest(Object sourceRequest) {
+    this.sourceRequest = sourceRequest;
+  }
```

Comment:
This property will make design not specific and not descriptive, I think this is not good extension.

Reply:
I don't think it's reasonable to treat the sourceRequest as a Object.
If it just a URI, we can change the Object to String.